pandas/core/dtypes/dtypes.py

```diff
@@ -479,8 +479,8 @@ def _is_boolean(self):
         return is_bool_dtype(self.categories)
 
 
-class DatetimeTZDtype(PandasExtensionDtype):
-
+@register_extension_dtype
+class DatetimeTZDtype(PandasExtensionDtype, ExtensionDtype):
```

Comment:
I *think* we can now move ``PandasExtensionDtype`` to subclass ``ExtensionDtype``, see the note in pandas.core.dtypes.base for _DtypeMixin

Reply:
Good as a followup item I think. IIRC I created an issue when I had to create PandasExtensionDtype in the first place.

Reply:
yes, i agree, do we have an issue about fixing this?